pkg/storage/kubernetes/client.go

```diff
@@ -110,6 +110,15 @@ func (i *Client) ListPods() ([]v1.Pod, error) {
 	return podEntries, nil
 }
 
+func (i *Client) GetPod(namespace, name string) (*v1.Pod, error) {
+	pod, err := i.clientSet.CoreV1().Pods(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return pod, nil
```

Comment:
hm, maybe I'm missing something, but why don't you just `return i.clientSet.CoreV1().Pods(namespace).Get(context.TODO(), name, metav1.GetOptions{})` ?